gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -108,26 +109,49 @@ From the command line, OpenShift procedures match Kubernetes procedures.
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 // end::reg-command-intro[]
 //
+// tag::k8s-connected-heading[]
+
+=== Connected Mode in Kubernetes
+
+// end::k8s-connected-heading[]
+// tag::openshift-connected-heading[]
+
+=== Connected Mode in OpenShift
+
+// end::openshift-connected-heading[]
+//
+// tag::k8s-local-heading[]
+
+=== Local Mode in Kubernetes
+
+// end::k8s-local-heading[]
+// tag::openshift-local-heading[]
+
+=== Local Mode in OpenShift
+
+// end::openshift-local-heading[]
 // tag::k8s-connected-intro[]
 
-In Connected Mode, Flex Gateway typically serves as an ingress or a standalone gateway that is managed mostly through Anypoint Platform, where you can: 
```

Comment:
@SebaElizalde Can you take a look at this (below) on Monday?

Reply:
we're merging into the beta site, but if you have issues with any rewording, we can fix before the release